packages/components/src/pages/VsCodeExtension.vue

```diff
@@ -401,16 +401,10 @@ export default {
       currentTraceFilterIndex: 0,
       showShareModal: false,
       shareURL: undefined,
+      shareEnabled: false,
     };
   },
 
-  props: {
-    appMapUploadable: {
-      type: Boolean,
-      default: false,
-    },
-  },
```

Comment:
`appMapUploadable` was already defaulted to `false` - what's the behavioral change this PR makes?